docs/contributing/source/ways-can-contribute/setup-dev-environment.md

```diff
@@ -0,0 +1,69 @@
+# Setup development environment
+
+## Setting up a single node development environment with ``docker-compose``
+
+You can also use [Docker Compose](https://docs.docker.com/compose/) to run all the tests.
+
+```bash
+$ docker-compose build bdb
+$ docker-compose up -d bdb
+```
+
+The above command will launch all 3 main required services/processes:
+
+* ``mongodb``
+* ``tendermint``
+* ``bigchaindb``
+
+To follow the logs of the ``tendermint`` service:
+
+```bash
+$ docker-compose logs -f tendermint
+```
+
+To follow the logs of the ``bigchaindb`` service:
+
+```bash
+$ docker-compose logs -f bigchaindb
+```
+
+To follow the logs of the ``mongodb`` service:
+
+```bash
+$ docker-compose logs -f mdb
+```
+
+Simple health check:
+
+```bash
+$ docker-compose up curl-client
```

Comment:
I thought the new `bdb` service was for healthcheck?

Reply:
yes, but `curl-client` is a separate service, it checks both tendermint and bighchaindb but you can also add. `bdb` is only for BigchainDB API .